File: ProgramingFundamentalsWithJava/LabsAndExercises/Arrays/EqualArrays.java
import java.util.Scanner;

public class EqualArrays {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String firstArraysString = scanner.nextLine();
        String[] firstNumbersStrings = firstArraysString.split(" ");
        int[] firstArray = new int[firstNumbersStrings.length];
        int sum = 0;
        for (int i = 0; i < firstArray.length; i++) {
            firstArray[i] = Integer.parseInt(firstNumbersStrings[i]);

        }
        String secondArraysString = scanner.nextLine();
        String[] secondNumbersStrings = secondArraysString.split(" ");
        int[] secondArray = new int[secondNumbersStrings.length];
        for (int i = 0; i < secondArray.length; i++) {
            secondArray[i] = Integer.parseInt(secondNumbersStrings[i]);

        }boolean equal = true;
        int differentAtIndex = -1;
        if (firstArray.length==secondArray.length){
            for (int i = 0; i < firstArray.length; i++) {
                if(firstArray[i]!=secondArray[i]){
                    equal = false;
                    differentAtIndex = i;
                    break;

                }
            }

        }else {
            equal = false;
        }if (equal){
            for (int number: firstArray) {
                sum+=number;
            }
            System.out.printf("Arrays are identical. Sum: %d", sum);
        }else {

            System.out.printf("Arrays are not identical. Found difference at %d index.",differentAtIndex );

        }

    }
}

File: ProgrammingFundamentalsWithJavaScript/LabsAndExercise/Arrays/Equal Arrays.js
function solve(a1, a2) {
    let sum = 0;
    let isTrue = true;
    let index;
    for (let i = 0; i < a1.length; i++) {
        if (a1[i] == a2[i]) {
            sum += Number(a1[i]);
        } else {
            isTrue = false;
            index = i;
            break;
        }
    }
    if (isTrue) {
        console.log('Arrays are identical. Sum: ' + sum);
    } else {
        console.log(`Arrays are not identical. Found difference at ${index} index`);
    }
    
} solve(['1'], ['10']);
